Cargo.toml

```diff
@@ -66,7 +66,11 @@ request-response = ["dep:libp2p-request-response"]
 rendezvous = ["dep:libp2p-rendezvous"]
 tcp = ["dep:libp2p-tcp"]
 uds = ["dep:libp2p-uds"]
-wasm-bindgen = ["futures-timer/wasm-bindgen", "instant/wasm-bindgen", "getrandom/js"]
+wasm-bindgen = [
+    "futures-timer/wasm-bindgen",
+    "instant/wasm-bindgen",
+    "getrandom/js",
+]
```

Comment:
Can we undo these formatting changes please?

Reply:
Sure thing, the formatter really wanted to do that one, but I'm ok with backing it out.

Reply:
We only use `rustfmt` in here so all you need is `cargo fmt`!

Unfortunately, it does not format manifest files yet so this must have been your editor :)

Reply:
Yes, I recently switched to vscode and everything has gone smoothly except for the formatter. It has given me problems in another file format in a python project. :rage4: